index.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title></title>
+  <link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+
+  <article>
+    <header>
+      <h3 class="cardtitle">Yellow Post-Its &reg</h3>
```

Comment:
Requirement 3 specified h1 tags, not h3. It's minute, but you seem to have all the other requirements correct, so good job!